backend/app/api/routes/scheduled_inquiries.py

```diff
@@ -0,0 +1,62 @@
+from fastapi import APIRouter, HTTPException
+
+from app.api.deps import SessionDep
+from app.models import ScheduledInquiryCreate, ScheduledInquiryPublic
+from app.models.scheduled_inquiry import ScheduledInquiriesPublic
+from app.services import (
+    inquiries as inquiries_service,
+)
+from app.services import (
+    scheduled_inquiries as scheduled_inquiries_service,
+)
+
+router = APIRouter()
+
+
+@router.post("/", response_model=ScheduledInquiryPublic)
+def create_scheduled_inquiry(
```

Comment:
I wonder if conceptually this isn't as much of a create operation as a "schedule an inquiry to be presented." I.e. we're not creating a scheduled inquiry, we're *scheduling* an inquiry. I get from the DB perspective it's an entry create, but it sounds a little weird from the overall conceptual perspective. Don't have a strong opinion either way, so feel free to ignore/resolve if you disagree.

Reply:
Yeah, I think that is a valid point. I had some internal conflict about creating separate modules for `scheduled_inquiries`. Would it make more sense to add endpoints for operations on scheduled inquiries to `api/routes/inquiries.py` instead, and update the signature to something like:
```py
@router.post("/scheduledinquiries", response_model=ScheduledInquiryPublic)
def schedule_inquiry ()
...
```
In this case, there would be a POST to `/inquiries/scheduled_inquiries` which might be weird. I'm honestly not sure whats best.

Reply:
I could also just keep the separation between `inquiries` and `scheduled_inquiries`, and update the name to something more that makes more sense conceptually.

Reply:
I like this solution more
/inquiries/{inquiry_id}/schedule
because you have the same base artifact, the Inquiry, and you are just adding an attribute to it

Reply:
yeah, I like that too.